content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```diff
@@ -556,7 +585,7 @@ section of the enhancement proposal about Pod topology spread constraints.
   cluster. This could lead to a problem in autoscaled clusters, when a node pool (or
   node group) is scaled to zero nodes, and you're expecting the cluster to scale up,
   because, in this case, those topology domains won't be considered until there is
-  at least one node in them.  
```

Comment:
Please retain the line break here (triggered by two spaces at the end of the line).

Reply:
If you'd like to insist this change, at least make it a separate commit (or even separate PR). It's not part of documenting `NodeInclusionPolicyInPodTopologySpread`.

Reply:
hold for https://github.com/kubernetes/website/pull/35966
/hold